src/Components/Auctions/RailSlider/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { SFC } from "react"
+import { graphql, QueryRenderer } from "react-relay"
+
+import { ContextConsumer, ContextProps, ContextProvider } from "../../Artsy"
+import { Slider } from "./Slider"
+
+interface Props extends ContextProps {
+  saleID: string
+}
+
+const SliderContainer = ContextConsumer((props: Props) => {
+  const { saleID, relayEnvironment } = props
+
+  return (
+    <QueryRenderer
+      environment={relayEnvironment}
+      query={graphql`
+        query RailSliderQuery($saleID: String!) {
+          sale(id: $saleID) {
+            ...Slider_sale
+          }
+        }
+      `}
+      variables={{ saleID }}
+      render={response => {
+        if (response.props) {
+          return <Slider sale={response.props.sale} />
+        } else {
+          return null
+        }
+      }}
+    />
+  )
+})
+
+export const RailSlider: SFC = () => {
+  return (
+    <ContextProvider>
+      <SliderContainer saleID="phillips-evening-and-day-editions-3" />
+    </ContextProvider>
```

Comment:
For follow-up work: A reason that this can’t really be inside this component is that the host (Force) is supposed to configure this context with e.g. the current user.